### ISSUE TITLE
Adding a basic GUI

### ISSUE THREAD

**Issue by ionicc** [ionicc/Youtube-mp3-downloader-light#10]
Addition of a basic GUI to help people who are not comfortable with cli. Any framework can be used (tkinter, kivy, qt, etc)

**Comment by Seeram** [comment]
I would like to do this in qt.

**Comment by ionicc** [comment]
That's great! I'll assign this issue to you 😄  @Seeram

**Comment by Seeram** [comment]
What's the process when I have assigned issue? Should I fork repo and make pull request?

**Comment by ionicc** [comment]
Yep, Just start the repo, Make a fork, Create a different branch with the addition you're making and make a pull request 😄

**Comment by crocarneiro** [comment]
I'll do it :)

**Comment by crocarneiro** [comment]
I'll do it :)

**Comment by crocarneiro** [comment]
So far I did this:

![gui](https://user-images.githubusercontent.com/10589421/47112170-ef98df00-d22b-11e8-858b-f303b8f79d86.gif)

It works for single videos. It lacks supports playlists and to cache the default path. What do you think?

**Comment by ionicc** [comment]
That's pretty great @crocarneiro  👍 
Although make sure to follow minimalism. As an user it should be really fast for you to open the script and start downloading. I'm pretty excited to see what you do for playlists 😄
